### PR TITLE
add RODARE publication file

### PR DESCRIPTION
**Pull request by psychocoderHPC**
This file is pushed to the branch release 2.5.0crp and will later merged back into the dev branch.

### CODE REVIEW

File: .rodare.json
Comment (sbastrakov) [inline .rodare.json:10]: Why didn't you put his affiliations?

Comment (psychocoderHPC) [reply .rodare.json:10]: I do not know it^^

Comment (psychocoderHPC) [reply .rodare.json:10]: The question is also should it be the old or new.

**Review by bernhardmgruber — APPROVED** [review]

File: .rodare.json
Comment (psychocoderHPC) [reply .rodare.json:10]: talked to axel it must be the old affiliatuon but can be optionally be extendet with "... now XYZ" but than we need to track the contributor so we should stay with the affiliarion from the date where the user contributed.

**Comment by psychocoderHPC** [comment]
updated @ax3l's affiliation and added @bernhardmgruber's orcid id

**Comment by psychocoderHPC** [comment]
@ax3l @slizzered @BenjaminW3 @Flamefire Please check your entry if the affiliation, from the time where you contributed, is fine.

File: .rodare.json
Comment (Flamefire) [inline .rodare.json:33]: If you like you can complete this with:

```suggestion
            "name": "Grund, Alexander"
            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
            "orcid": "0000-0002-7196-8452"
```

File: .rodare.json
Comment (slizzered) [inline .rodare.json:5]:
```suggestion
            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden",
            "orcid": "0000-0002-6459-0842"
```

Comment (slizzered) [reply .rodare.json:5]: If you want, it would be cool if you added my orcid-ID as well :)

Comment (psychocoderHPC) [reply .rodare.json:5]: @slizzered Thx for the fedback, sure I will add your ORCID

**Comment by psychocoderHPC** [comment]
update the file, can be merged after CI passed

**Comment by psychocoderHPC** [comment]
@alpaka-group/mallocmc-maintainers  This can be merged

**Review by sbastrakov — APPROVED** [review]

**Comment by ax3l** [comment]
@psychocoderHPC since mallocMC is already [releasing to Zenodo](https://doi.org/10.5281/zenodo.591745) since many releases, shouldn't this be a `.zenodo.json` file?

**Comment by psychocoderHPC** [comment]
> @psychocoderHPC since mallocMC is already [releasing to Zenodo](https://doi.org/10.5281/zenodo.591745) since many releases, shouldn't this be a `.zenodo.json` file?

Until last week I was not aware that it is on zenodoo because there is no yaml file in and I haven't checked the old release. 
Maybe I will change the yaml to a zenodoo file.

**Comment by ax3l** [comment]
Yep, there is no `.zenodo.json ` in this repo because we did it manually so far. But as you see on the release page it is already active, so let's avoid a consistency break.

**Pull request merged**
